source/cpu/security_amd64.go

```diff
@@ -52,3 +59,16 @@ func sgxEnabled() bool {
 
 	return false
 }
+
+func tdxEnabled() bool {
+	// If /sys/module/kvm_intel/parameters/tdx is not present, or is present
+	// with a value different than "Y\n" assume TDX to be unavailable or
+	// disabled.
+	protVirtHost := source.SysfsDir.Path("module/kvm_intel/parameters/tdx")
+	if content, err := os.ReadFile(protVirtHost); err == nil {
```

Comment:
Do we want to print a warning/error if the file exists but we cannot read it? I.e. smth like

```
if ! os.IsNotExist(err) {
    klog.Warning(...)
}

Dunno, just asking...

Reply:
I am not sure.

Taking a look at other places we do similar things (looking specifically at s390x here), we don't do such a check.
I'd leave it as it's for now, if you don't mind.

Reply:
That's fine I think